### PR TITLE
Document trezor-core event loop

### PR DESCRIPTION
**Pull request by matejcik**
There was some gap between me learning about the event loop and finishing the documentation, so things might be missing that are obvious to me.

Please let me know what you think.

Also this should probably be linked from some index - which?

### CODE REVIEW

File: docs/core/src/event-loop.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+## Low-level API
+
+### Function summary
+
+`loop.run()` starts the event loop. The call only returns when there are no further
+waiting tasks -- so, in usual conditions, never.
+
+`loop.schedule(task, value, deadline, finalizer)` schedules an awaitable to be run
+either as soon as possible, or at a time specified by `deadline`, in microseconds since
+system bootup.
```
[the remark this answers is not in the document]
Comment (matejcik) [reply docs/core/src/event-loop.md:32]: exactly. it's "_at a time_ specified by `deadline`", not "_after_ a delay"

**Comment by tsusanka** [comment]
> Also this should probably be linked from some index - which?

I think we can start to build some general info in docs/core/index.html?

Also, please add a link to SUMMARY.md to this document, so mdbook lists it in the left column.

File: docs/core/src/event-loop.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+is not done immediately. Instead, the `wait` begins on the next tick (or whenever the
+newly created coroutine runs) and the `sleep` in the tick afterwards. When nesting
+multiple `race`s, the child `races` also run later.
+
+Also, when a child task is done, another scheduling gap happens, and the parent task
+is scheduled to run on the next tick.
+
+_Upcoming changes may solve this in relevant cases, by inlining syscall operations._
+
+**`loop.chan()`** is a two-way communication object that actually implements two
```
Comment (jpochyla) [inline docs/core/src/event-loop.md:186]: `chan` is one-way, no?

**Comment by matejcik** [comment]
the fixup commit should resolve all comments so far

**Pull request merged**